frontend/app/components/input-select.tsx

```diff
@@ -0,0 +1,79 @@
+import { type ComponentProps, type ReactNode, forwardRef } from 'react';
+
+import clsx from 'clsx';
+
+import { InputError } from './input-error';
+import { InputHelp } from './input-help';
+import { InputLabel } from './input-label';
+import { InputOption } from './input-option';
```

Comment:
Should these be aliased imports (`~/components/`)?

Maybe we should have a discussion about this, because I feel like in some situations, using `./` makes more sense.

Reply:
Since the components in question are closely related (and in the same directory) as the `InputSelect` component, I suggest leaving these lines as is.

Reply:
Yeah.. I wonder if we should create a `~/components/form` or similar folder to hold all of the form inputs?

Also, just noticed that `<InputOption>` is in its own file. Would you ever use this outside of the `<InputSelect>` component? If not, why not just move it inside?

Reply:
> Yeah.. I wonder if we should create a `~/components/form` or similar folder to hold all of the form inputs?

I'm okay with that (in a future PR)

> Also, just noticed that `<InputOption>` is in its own file. Would you ever use this outside of the `<InputSelect>` component? If not, why not just move it inside?

My thinking was that if we ever need to use an `InputOption` inside some other element, like components to render a `<optgroup>` or `<datalist>`, having a separate `InputOption` file gives us this flexibility.

Reply:
I didn't think about those use cases. I'll resolve this thread.